openmmtools/tests/test_cache.py

```diff
@@ -218,10 +218,12 @@ def test_copy_integrator_state(self):
         def read_attribute(integrator, attribute_name):
             try:
                 return getattr(integrator, attribute_name)
-            except:
+            except Exception as err:
+                raise err
                 try:
                     return getattr(integrator, 'get' + attribute_name)()
-                except:
+                except Exception as err:
+                    raise err
```

Comment:
I think we just want to check for `AttributeError` here, in both cases.

Reply:
Yep! I wanted to see what sorts of error(s) they could be and I agree!